docs/using/recipes.md

```diff
@@ -528,7 +528,7 @@ Please note that the [Delta Lake](https://delta.io/) packages are only available
 By adding the properties to `spark-defaults.conf`, the user no longer needs to enable Delta support in each notebook.
 
 ```dockerfile
-FROM jupyter/pyspark-notebook:latest
+FROM jupyter/pyspark-notebook:lab-3.1.18
 
 ARG DELTA_CORE_VERSION="1.0.0"
```

Comment:
It seems that there is a release newer release of delta lake, which supports newer spark.
https://github.com/delta-io/delta/releases/tag/v1.1.0

So, let's bump version here

Reply:
In that case if the decision is to go with the latest version of both the DELTA_CORE_VERSION should be changed to 1.1.0. 

```suggestion
ARG DELTA_CORE_VERSION="1.1.0"
```